app/controllers/transcriptions_controller.rb

```diff
@@ -24,6 +26,27 @@ def update_attrs
     jsonapi_deserialize(params)
   end
 
+  # jsonapi.rb filtering doesn't handle filtering by enum term (e.g. 'ready'),
+  # so we must translate status terms to their integer value if they're present
+  def status_filter_to_int
+    if params[:filter]
```

Comment:
minor, not blocking - you could avoid the conditional here by wrapping in an array
```
    Array.wrap(params[:filter]).each
```

Reply:
Interesting, I haven't tried doing this before, thanks for tip. Going to stick with as is because front end wants this fix in asap, and I'd rather not spend time testing to make sure this change works 😅